popup.css

```
@@ -1,815 +1,443 @@
+  border-color: #ffac8c;
+  background: linear-gradient(130deg, #a8442f, #cf603d);
+  box-shadow: 0 0 0 3px rgba(219, 116, 76, 0.22);
+}
+
+.btn-icon {
+  display: inline-grid;
+  width: 30px;
+  height: 30px;
+  place-items: center;
+  margin-bottom: var(--space-1);
+  border: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)